### PR TITLE
feat(pie-docs): DSW-1884 remove /overview in component page urls 

### PR DESCRIPTION
**Pull request by jamieomaguire**
## Describe your changes (can list changeset entries if preferable)
---
"pie-docs": minor
---

[Changed] - Removed /overview urls from components pages and use overview content as the index.html of each directory. I.e. /components/button for the overview page instead of /components/button/overview

---
"@justeattakeaway/pie-icon-button": patch
"@justeattakeaway/pie-divider": patch
"@justeattakeaway/pie-button": patch
"@justeattakeaway/pie-switch": patch
"@justeattakeaway/pie-modal": patch
"@justeattakeaway/pie-card": patch
"@justeattakeaway/pie-chip": patch
"@justeattakeaway/pie-link": patch
"@justeattakeaway/pie-tag": patch
---

[Changed] - Point to updated component page link in docs site from readme file

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have added thorough tests where applicable (unit / component / visual)
- [x] I have reviewed the Docs site preview
- [x] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1 - @xander-marjoram 
- [x] I have reviewed the Docs site preview
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2 - @leksaBoiko 
- [x] I have reviewed the Docs site preview
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: apps/pie-docs/test/helpers/routes-helper.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const readChildren = (childDirectories, result = []) => {
     // This allows us to include urls such as /foundations, /foundations/typography, etc.
     // Some of these pages should display a 404 page but we still want to include them in our routes as they technically exist.
     const hasIndexHtmlChild = childDirectories.children.some((child) => child.name === 'index.html');
-    const directoriesToInclude = ['foundations'];
+    const directoriesToInclude = ['foundations', 'components'];
```
Comment (xander-marjoram) [inline apps/pie-docs/test/helpers/routes-helper.js:40]: Nice

File: apps/pie-docs/src/components/template/template.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 eleventyNavigation:
     key: Template
     parent: Components
-    # url: /components/template/overview/  // should be uncomented for a live component page
+    # url: /components/template/  // should be uncomented for a live component page
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/template/template.md:5]: Please could you copy this comment to the other lines that should be uncommented?
```suggestion
    # url: /components/template/  // should be uncommented for a live component page
```

File: apps/pie-docs/src/components/icon-button/overview/overview.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ eleventyNavigation:
 eleventyComputed:
     sizes: "{% include './sizes.json' %}"
 shouldShowContents: true
+permalink: /components/icon-button/
```
Comment (leksaBoiko) [inline apps/pie-docs/src/components/icon-button/overview/overview.md:9]:
```suggestion
permalink: components/icon-button/
```

Comment (jamieomaguire) [reply apps/pie-docs/src/components/icon-button/overview/overview.md:9]: ta!

**Review by xander-marjoram — APPROVED** [review]

**Review by maledr5 — APPROVED** [review]

**Pull request merged**